Mecrano/Personal-Page
# feat: :lipstick: create basic project card implementation

#### What problem is this solving?

Project Card view on home

#### How to test it?

1. Run `yarn dev`
2. Go to browser and run dataLayer in the console section
3. You now should see the events fired to analitycs
